### PR TITLE
Extract "assert_instance_command" method

### PR DESCRIPTION
**Pull request by medb**

### CODE REVIEW

File: integration_tests/dataproc_test_case.py [hunk 1]
```diff
@@ -146,14 +146,21 @@ def getClusterName(self):
...
-        self.run_and_assert_command('gcloud compute scp {} {}:'.format(
-            testfile, name))
+        self.assert_command('gcloud compute scp {} {}:'.format(testfile, name))
 
     def remove_test_script(self, testfile, name):
-        self.run_and_assert_command(
-            'gcloud compute ssh {} --command="rm {}"'.format(name, testfile))
+        self.assert_instance_command(name, "rm {}".format(testfile))
+
+    def assert_instance_command(self,
```
Comment (functicons) [inline integration_tests/dataproc_test_case.py]: Add doc for this method.

File: integration_tests/dataproc_test_case.py [hunk 2]
```diff
@@ -146,14 +146,21 @@ def getClusterName(self):
...
+                                instance,
+                                cmd,
+                                timeout_in_minutes=DEFAULT_TIMEOUT):
+        ret_code, stdout, stderr = self.assert_command(
+            'gcloud compute ssh {} --command="{}"'.format(instance, cmd),
+            timeout_in_minutes)
+        return ret_code, stdout, stderr
 
-    def run_and_assert_command(self, cmd, timeout_in_minutes=DEFAULT_TIMEOUT):
+    def assert_command(self, cmd, timeout_in_minutes=DEFAULT_TIMEOUT):
```
Comment (functicons) [inline integration_tests/dataproc_test_case.py]: ditto.

File: integration_tests/dataproc_test_case.py (hunk 1 above)
Comment (medb) [reply integration_tests/dataproc_test_case.py]: Done.

File: integration_tests/dataproc_test_case.py (hunk 2 above)
Comment (medb) [reply integration_tests/dataproc_test_case.py]: Done.

**Pull request merged**